tests/invalid_models_tests/test_relative_fields.py

```diff
@@ -89,6 +89,36 @@ class Model(models.Model):
         field = Model._meta.get_field("m2m")
         self.assertEqual(field.check(from_model=Model), [])
 
+    def test_many_to_many_with_limit_choices_auto_created_no_warning(self):
+        class Model(models.Model):
+            name = models.CharField(max_length=20)
+
+        class ModelM2M(models.Model):
+            m2m = models.ManyToManyField(Model, limit_choices_to={"name": "test_name"})
+
+        self.assertEqual(ModelM2M.check(), [])
```

Comment:
This test is unnecessary, other tests will fail if we raise a warning for `ManyToManyField` without `db_comment`.